### PR TITLE
Added support for Switchbot

### PR DESCRIPTION
**Pull request by AS137430**
# Description

Added support for Switchbot. The worker code was modified from thermostat.py as it has on_command example. [This](https://www.switch-bot.com/) is a switch that uses BLE to trigger a rocker switch or button on/off. Dependencies that are required for this change: binascii.  

## Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

### CODE REVIEW

**Comment by AS137430** [comment]
Home Assistant configuration.yaml example

```
- platform: mqtt
    name: "Heater Switch"
    state_topic: "switchbot/bathroom/heater"
    command_topic: "switchbot/bathroom/heater/set"
    availability_topic: "switchbot/bathroom/heater/available"
    payload_on: "ON"
    payload_off: "OFF"
    state_on: "ON"
    state_off: "OFF"
    optimistic: false
    qos: 1
    retain: false
```

**Comment by bbbenji** [comment]
Awesome! Looks good from my side.

File: workers/switchbot.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      _LOGGER.debug("Updating %s device '%s' (%s)", repr(self), name, bot["mac"])
+      try:
+        ret += self.update_device_state(name, bot["state"])
+      except btle.BTLEException as e:
+        logger.log_exception(_LOGGER, "Error during update of %s device '%s' (%s): %s", repr(self), name, bot["mac"], type(e).__name__, suppress=True)
+    return ret
+
+  def on_command(self, topic, value):
+    from bluepy import btle
+    import binascii
```
Comment (zewelor) [inline workers/switchbot.py]: Please add it to REQUIREMENTS array.

**Comment by zewelor** [comment]
Great !

**Comment by AS137430** [comment]
Thanks. I cannot seem to figure out how to edit the wiki to add in the configuration.yaml details for Switchbot. Can anybody help?

**Comment by bbbenji** [comment]
https://github.com/zewelor/bt-mqtt-gateway/wiki/Home-Assistant/_edit

**Comment by AS137430** [comment]
I don't have the permission to edit

**Comment by AS137430** [comment]
Thanks for editing. @bbbenji. Can you add the following as well to make it clear to users?

For an explanation of this config see the [MQTT Switch](https://www.home-assistant.io/components/switch.mqtt/) page on the HASS Wiki.

**Pull request merged**